src/tests/system/tests/test_sss_override.py

```diff
@@ -0,0 +1,246 @@
+from __future__ import annotations
+
+import pytest
+from sssd_test_framework.roles.client import Client
+from sssd_test_framework.roles.generic import GenericProvider
+from sssd_test_framework.topology import KnownTopology
+
+
+@pytest.mark.topology(KnownTopology.LDAP)
+@pytest.mark.topology(KnownTopology.AD)
```

Comment:
All tests here can also run against Samba (which is important so they run inside PR CI) and I believe it can also be used with IPA.